### PR TITLE
fixes #3249

### LINKED ISSUE #3249: Gas miners don't turn on after being unwrenched and then wrenched
I tried every tool, nothing, couldn't turn it on.

### PR DESCRIPTION
**Pull request by Intigracy**

### CODE REVIEW

File: code/game/machinery/atmoalter/gas_mine.dm [hunk 1]
```diff
@@ -45,6 +45,10 @@
 
 /obj/machinery/atmospherics/miner/power_change()
 	..()
+	if( powered() )
```
Comment (PJB3005) [inline code/game/machinery/atmoalter/gas_mine.dm]: Isn't this handled in `/obj/machinery/proc/power_change()`?

Comment (Intigracy) [inline code/game/machinery/atmoalter/gas_mine.dm]: yes, oops

File: code/game/machinery/atmoalter/gas_mine.dm [hunk 2]
```diff
@@ -54,12 +54,10 @@
...
-		update_icon()
-
-/obj/machinery/atmospherics/miner/attack_ai(var/mob/user)
-	..()
-	on=!on
-	update_icon()
+		power_change()
+		to_chat(user, "<span class='warning'>You toggle \the [src] [on ? "off" "on"].</span>")
+	else
+		to_chat(usr, "<span class='warning'>That needs to be bolted to the ground first.</span>")
```
Comment (PJB3005) [inline code/game/machinery/atmoalter/gas_mine.dm]: `user`

File: code/game/machinery/atmoalter/gas_mine.dm [hunk 3]
```diff
@@ -54,12 +54,10 @@
 	..()
 	if(anchored)
 		on=!on
-		update_icon()
-
-/obj/machinery/atmospherics/miner/attack_ai(var/mob/user)
```
Comment (Intigracy) [inline code/game/machinery/atmoalter/gas_mine.dm]: parent of attack_ai on machines calls attack_hand, zero reason to change the child here

File: code/game/machinery/atmoalter/gas_mine.dm (hunk 2 above)
Comment (PJB3005) [inline code/game/machinery/atmoalter/gas_mine.dm]: Also IMO for consistency you should use `\the [src]` instead of `that`.

Correction: `\The`

Comment (Intigracy) [inline code/game/machinery/atmoalter/gas_mine.dm]: does \the auto capitalize?

Comment (PJB3005) [inline code/game/machinery/atmoalter/gas_mine.dm]: If you use `\The` then you get it capitalized, `\the` is not capitalized.

**Comment by ghost** [comment]
The irony is this won't close that issue as you didn't reference it in the OP

**Comment by Intigracy** [comment]
Commit messages work too you moron.

**Comment by ghost** [comment]
Commit messages commit schmessages

**Comment by clusterfack** [comment]
I think you need to put in an adjacency check for this. In the past that is how it worked and I think for a good reason, considering getting to and from gas miners can be a pain.

**Comment by clusterfack** [comment]
Er actually no they didn't have an adjacency check, it had a !anchored requirement to turn on/off, which I guess was also somewhat weird but it prevented borgs from just up and turning off the gas miners

**Comment by Intigracy** [comment]
So adjacency check, yes/no?

**Comment by clusterfack** [comment]
Yeah I'd say do it

**Comment by Intigracy** [comment]
k done

**Pull request merged**